services/exam-manager/app/db.py

```diff
@@ -128,10 +128,14 @@ class Task(Base):
     # Input and output artifacts with export destination
     # Example: {"input": [{"name": "env_HOLOSCAN_INPUT_PATH", "value": "{{ context.input.dicom }}"}]}
     # https://github.com/Project-MONAI/monai-deploy/blob/main/deploy/monai-deploy-express/sample-workflows/hello-world.json
-    artifacts: Mapped[dict[str, list[dict[str, str]]]] = mapped_column(type_=JSON, nullable=True) # implemention of input and output artifacts for artifact types as replacement for string
-
-    # List of task destinations, which are for example used to create the chain of topics in the kafka message broker, i.e., target topics
-    task_destinations: Mapped[list[dict[str, str]]] = mapped_column(type_=JSON, nullable=True)
+    # implemention of input and output artifacts for artifact types as replacement for string
+    artifacts: Mapped[dict[str, str]] = mapped_column(type_=JSON, nullable=True)
+    # artifacts: Mapped[dict[str, list[dict[str, str]]]] = mapped_column(type_=JSON, nullable=True)
+
+    # List of task destinations, which are for example used to create the chain of topics in the kafka message broker,
+    # i.e., target topics
+    destinations: Mapped[dict[str, str]] = mapped_column(type_=JSON, nullable=True)
+    # task_destinations: Mapped[list[dict[str, str]]] = mapped_column(type_=JSON, nullable=True)
```

Comment:
remove?

Reply:
I can update it later today, but feel free to delete the commented lines 👍🏼